lib/config/themes.dart

```diff
@@ -200,6 +207,22 @@ abstract class FluffyThemes {
               ? LinagoraSysColors.material().primary
               : LinagoraSysColors.material().primaryDark,
         ),
+        iconButtonTheme: IconButtonThemeData(
+          style: ButtonStyle(
+            iconSize: MaterialStateProperty.all(iconSize),
+            iconColor: MaterialStateProperty.all(
+              brightness == Brightness.light
+                ? Colors.black
+                : Colors.white
```

Comment:
IMO, no black/white in design. why not use 
<img width="419" alt="image" src="https://github.com/linagora/twake-on-matrix/assets/6462404/b9617444-11a4-4026-9417-247f5b423fde">